sdk/metrics/exporter/exporter.go

```diff
@@ -0,0 +1,52 @@
+package exporter
+
+import (
+	"context"
+
+	"go.opentelemetry.io/api/core"
+	api "go.opentelemetry.io/api/metric"
+)
+
+type (
```

Comment:
I know this is still WIP. But shouldn't this be in the `export` package?

Reply:
Done